corehq/apps/events/templates/partials/attendee_conversion_status.html

```diff
@@ -0,0 +1,15 @@
+{% extends "hqwebapp/partials/download_status.html" %}
+{% load i18n %}
+{% block results %}
+  <div class="alert alert-success">
+    <p>{{ on_complete_long }}</p>
+  </div>
+  {% if result.errors %}
```

Comment:
I don't see `errors` in the task getting set above, may be the DownloadProgress sets that, but in any case, have you tested by failing the task and see this error popup?

Reply:
@sravfeyn Good question. I think the task should actually set and return the errors in the task,
```
return {'messages': {'errors': [...]}}
```
but we don't do that, so that piece of HTML is redundant.

Looking at the task I don't think there's anything worth reporting to the user such that the user can take any action to rectify the said error, so I'm going to remove that piece of HTML.